### PR TITLE
Regenerate Face data-plane SDK 0.5.0

### PR DESCRIPTION
**Pull request by tjprescott**
Adds Mask model and associated properties.

### CODE REVIEW

File: sdk/cognitiveservices/azure-cognitiveservices-vision-face/azure/cognitiveservices/vision/face/version.py [hunk 1]
```diff
@@ -9,5 +9,5 @@
 # regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "0.4.1"
+VERSION = "0.5.0"
```
Comment (tjprescott) [inline sdk/cognitiveservices/azure-cognitiveservices-vision-face/azure/cognitiveservices/vision/face/version.py:12]: @lmazuel I'm not sure if this should be "0.5.0" or "0.4.2"

Comment (lmazuel) [reply sdk/cognitiveservices/azure-cognitiveservices-vision-face/azure/cognitiveservices/vision/face/version.py:12]: Let's make it 0.5.0

**Review by rakshith91 — COMMENTED** [review]
Please update the changelog

File: sdk/cognitiveservices/azure-cognitiveservices-vision-face/README.md [hunk 2]
```diff
@@ -7,15 +7,17 @@ For a more complete view of Azure libraries, see the [azure sdk python release](
...
+
+For code examples, see [Cognitive Services Face](https://docs.microsoft.com/python/api/overview/azure/cognitive-services) on docs.microsoft.com.
 
 
 # Provide Feedback
 
 If you encounter any bugs or have suggestions, please file an issue in the
 [Issues](https://github.com/Azure/azure-sdk-for-python/issues)
-section of the project.
+section of the project.
```
Comment (rakshith91) [inline sdk/cognitiveservices/azure-cognitiveservices-vision-face/README.md:20]:
```suggestion
section of the project.
```

Comment (tjprescott) [reply sdk/cognitiveservices/azure-cognitiveservices-vision-face/README.md]: This typo seems to be automatically added by automation. I made the suggested change though.

File: sdk/cognitiveservices/azure-cognitiveservices-vision-face/README.md [hunk 3]
```diff
@@ -7,8 +7,10 @@ For a more complete view of Azure libraries, see the [azure sdk python release](
 
 # Usage
 
-For code examples, see [Cognitive Services Face](https://docs.microsoft.com/python/api/overview/azure/cognitive-services)
-on docs.microsoft.com.
+
+
+
```
Comment (tjprescott) [inline sdk/cognitiveservices/azure-cognitiveservices-vision-face/README.md:12]:
```suggestion
```

**Comment by tjprescott** [comment]
@lmazuel I added the changelog so this should be ready to merge.

**Review by lmazuel — APPROVED** [review]

**Pull request merged**